src/loader.c

```diff
@@ -198,17 +227,20 @@ static void *imv_loader_bg_new_img(void *data)
       raw_frame_time = *(int*)FreeImage_GetTagValue(tag);
     }
     FreeImage_UnlockPage(mbmp, frame, 0);
-
   } else {
     /* Future TODO: If we load image line-by-line we could stop loading large
      * ones before wasting much more time/memory on them. */
-    FIBITMAP *image = FreeImage_Load(fmt, path, 0);
-    free(path);
-    if(!image) {
-      imv_loader_error_occurred(ldr);
-      return 0;
+    FIBITMAP *image = NULL;
+    if(load_from_memory != 1){
```

Comment:
Also, space between `)` and `{` on ifs please, for consistency with the rest of the code.